### ISSUE TITLE
Building OpenLiberty 19.0.0.8 Fails

### ISSUE THREAD

**Issue by FireBurn** [OpenLiberty/open-liberty#8557]
Building with OpenJDK 8.222

./gradlew cnf:initialize
./gradlew assemble

I can compress and attache the full log if you like, but uncompressed it's 5MB+

```
> Task :com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6:compileJava FAILED
/apps/was/jenkins/workspace/Compile_OpenLiberty/open-liberty/dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/src/org/apache/cxf/binding/soap/saaj/SAAJStreamWriter.java:41: error: cannot find symbol
import static org.apache.cxf.binding.soap.saaj.SAAJUtils.adjustPrefix;
^
  symbol:   static adjustPrefix
  location: class
/apps/was/jenkins/workspace/Compile_OpenLiberty/open-liberty/dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/src/org/apache/cxf/binding/soap/saaj/SAAJStreamWriter.java:113: error: cannot find symbol
                adjustPrefix((Element)nd2, pfx);
                ^
  symbol:   method adjustPrefix(Element,String)
  location: class SAAJStreamWriter
/apps/was/jenkins/workspace/Compile_OpenLiberty/open-liberty/dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/src/org/apache/cxf/binding/soap/saaj/SAAJStreamWriter.java:115: error: cannot find symbol
                    adjustPrefix(getEnvelope().getHeader(), pfx);
                    ^
  symbol:   method adjustPrefix(SOAPHeader,String)
  location: class SAAJStreamWriter
/apps/was/jenkins/workspace/Compile_OpenLiberty/open-liberty/dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/src/org/apache/cxf/binding/soap/saaj/SAAJStreamWriter.java:133: error: cannot find symbol
                    setChild(adjustPrefix(getEnvelope(), prefix), false);
                             ^
  symbol:   method adjustPrefix(SOAPEnvelope,String)
  location: class SAAJStreamWriter
/apps/was/jenkins/workspace/Compile_OpenLiberty/open-liberty/dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/src/org/apache/cxf/binding/soap/saaj/SAAJStreamWriter.java:134: error: cannot find symbol
                    adjustPrefix(getEnvelope().getHeader(), prefix);
                    ^
  symbol:   method adjustPrefix(SOAPHeader,String)
  location: class SAAJStreamWriter
/apps/was/jenkins/workspace/Compile_OpenLiberty/open-liberty/dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/src/org/apache/cxf/binding/soap/saaj/SAAJStreamWriter.java:135: error: cannot find symbol
                    adjustPrefix(getEnvelope().getBody(), prefix);
                    ^
  symbol:   method adjustPrefix(SOAPBody,String)
  location: class SAAJStreamWriter
/apps/was/jenkins/workspace/Compile_OpenLiberty/open-liberty/dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/src/org/apache/cxf/binding/soap/saaj/SAAJStreamWriter.java:143: error: cannot find symbol
                    setChild(adjustPrefix(getEnvelope().getBody(), prefix), false);
                             ^
  symbol:   method adjustPrefix(SOAPBody,String)
  location: class SAAJStreamWriter
/apps/was/jenkins/workspace/Compile_OpenLiberty/open-liberty/dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/src/org/apache/cxf/binding/soap/saaj/SAAJStreamWriter.java:149: error: cannot find symbol
                    setChild(adjustPrefix(getEnvelope().getHeader(), prefix), false);
                             ^
  symbol:   method adjustPrefix(SOAPHeader,String)
  location: class SAAJStreamWriter
/apps/was/jenkins/workspace/Compile_OpenLiberty/open-liberty/dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/src/org/apache/cxf/binding/soap/saaj/SAAJStreamWriter.java:162: error: cannot find symbol
                    setChild(adjustPrefix(f, prefix), false);
                             ^
  symbol:   method adjustPrefix(SOAPFault,String)
  location: class SAAJStreamWriter
/apps/was/jenkins/workspace/Compile_OpenLiberty/open-liberty/dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/src/org/apache/cxf/binding/soap/saaj/SAAJStreamWriter.java:221: error: cannot find symbol
                    adjustPrefix(el, pfx);
                    ^
  symbol:   method adjustPrefix(Element,String)
  location: class SAAJStreamWriter
10 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6:compileJava'.
> Compilation failed; see the compiler error output for details.
```

**Comment by neuwerk** [comment]
@FireBurn Hey, thanks for opening an issue for this. I've been looking into it since yesterday, and in the process of chasing down the issue I was able to recreate the issue you're seeing. Then I tried running `./gradlew assemble` with Java 11 and I was able to build successfully, so if you need a temporary work around I suggest you try changing your JDK and rebuilding.

**Comment by FireBurn** [comment]
Can the resulting build work with jdk8 or does it force everything to need 11 after that?

**Comment by neuwerk** [comment]
Nope, changing JDKs after building is just fine. After you switch JDKs, run `./gradlew --stop`  prior to building to ensure your gradle daemon picks up the JDK change.

**Comment by FireBurn** [comment]
Same for 19.0.0.9
[log.zip](https://github.com/OpenLiberty/open-liberty/files/3583859/log.zip)

**Comment by neuwerk** [comment]
@FireBurn I'm so sorry for the delay getting back to you, I've been out for a while. I originally was able to recreate the problem you were having, but it was only occasionally. I hadn't gotten to the point of finding a root-cause, but we did a major refactor of this code within the last couple of weeks so that change will be included in 19.0.0.11. Since the refactor, I have not been able to recreate the issue at all. So I'm not sure if the refactor provided an incidental fix, or if I'm not able to recreate the failure. I'll keep digging in the mean-time, but out of curiosity can you recreate the issue in 19.0.0.10?

**Comment by FireBurn** [comment]
I can confirm that 19.0.0.10 builds fine with OpenJDK 8 again, thanks
